docs/root/configuration/upstream/cluster_manager/cluster_stats.rst

```diff
@@ -176,7 +176,7 @@ Circuit breakers statistics will be rooted at *cluster.<name>.circuit_breakers.<
 Timeout budget statistics
 -------------------------
 
-If :ref:`timeout budget statistic tracking <envoy_v3_api_field_config.cluster.v3.Cluster.track_timeout_budgets>` is
+If :ref:`timeout budget statistic tracking <envoy_v3_api_field_config.cluster.v3.Cluster.track_cluster_stats>` is
 turned on, statistics will be added to *cluster.<name>* and contain the following:
```

Comment:
should we keep documentation for deprecated field as well during first deprecation cycle? (cc @htuch )

Reply:
confirmed with @htuch, documentation for deprecated field should still be in place @ranjithkumar007

Reply:
Will update